src/quo2/components/browser/browser_input/view.cljs

```diff
@@ -0,0 +1,145 @@
+(ns quo2.components.browser.browser-input.view
+  (:require [quo2.components.icon :as icon]
+            [quo2.components.browser.browser-input.style :as style]
+            [react-native.core :as rn]
+            [reagent.core :as reagent]
+            [quo2.foundations.colors :as colors]
+            [quo2.theme :as quo2.theme]
```

Comment:
just `theme`, quo2 is redundant

Reply:
actually it's not @flexsurfer. Here we are using `theme` prop as well so there is a likely chance these two references clash and there is unsafe uses. 
Similar to @ilmotta's pr: https://github.com/status-im/status-mobile/pull/16500

however we have been using `[quo2.theme :as quo.theme]` in other files